apps/admin-gui/src/app/shared/components/dialogs/add-edit-notification-dialog/add-edit-notification-dialog.component.html

```diff
@@ -120,23 +121,26 @@ <h1 mat-dialog-title>{{'DIALOGS.NOTIFICATIONS_ADD_EDIT_MAIL.TITLE_EDIT' | transl
                 <div class="font-weight-bold">
                   {{'DIALOGS.NOTIFICATIONS_ADD_EDIT_MAIL.TEXT' | translate}}:
                 </div>
-                <button mat-flat-button (click)="this.showTags = !this.showTags;">
-                  {{showTags ? ('DIALOGS.NOTIFICATIONS_ADD_EDIT_MAIL.HIDE_TAGS' | translate) :
-                  ('DIALOGS.NOTIFICATIONS_ADD_EDIT_MAIL.SHOW_TAGS' | translate)}}
-                </button>
+
                 <mat-form-field class="w-100">
                 <textarea matInput
                           #Textarea
-                          rows="17"
+                          rows="10"
```

Comment:
so, I would git here more rows than 10, but we will discuss this and everything else on daily scrum